Vulkan-Pipeline-Assistant/mainwindow.cpp

```diff
@@ -418,62 +449,29 @@ namespace vpa {
     }
 
     void MainWindow::MakeDescriptorBlock() {
-        QWidget*& container = m_configBlocks[m_descriptorBlockIdx];
-        if (container) delete container;
-        container = new QWidget(m_rightTopContainer);
-        QGridLayout* layout = new QGridLayout(container);
-        layout->setColumnStretch(0, 1);
-        layout->setColumnStretch(1, 4);
-        layout->setAlignment(Qt::AlignTop);
-        container->setLayout(layout);
-
-        QWidget* leftGroup = new QWidget(container);
-        QWidget* rightGroup = new QWidget(container);
-        leftGroup->setLayout(new QVBoxLayout(leftGroup));
-        rightGroup->setLayout(new QVBoxLayout(rightGroup));
-        layout->addWidget(leftGroup, 0, 0);
-        layout->addWidget(rightGroup, 0, 1);
-        leftGroup->layout()->setAlignment(Qt::AlignTop);
-        rightGroup->layout()->setAlignment(Qt::AlignTop);
-
         if (m_vulkan) {
             Descriptors* descriptors = m_vulkan->GetDescriptors();
+            m_descriptorDrawer->Clear();
             if (descriptors) {
```

Comment:
if not descriptors, return early instead of wrapping the code

Reply:
I agree with this, updated.